origami.json

```diff
@@ -48,6 +48,13 @@
 			"hidden": true,
 			"description": ""
 		},
+		{
+			"title": "Approved Alternative Cookie Banner with Privacy Policy",
+			"name": "privacy-policy-cookie-message",
+			"template": "demos/src/privacy-policy-cookie-message.mustache",
+			"hidden": false,
+			"description": "This cookie banner is a visual variant on the approved version for all FT products and contains links back to the FT's cookie policy. The copy in the banner has been approved by the FT's legal team."
```

Comment:
Should we add in a note on not using this for FT.com things here? Or possibly just make it a hidden demo, you can still send people directly to the demo page if they need it

Reply:
I'll do both 👍